app/views/embed/footer/_generic.html.erb

```diff
@@ -10,8 +10,21 @@
         data-sul-embed-toggle= 'sul-embed-embed-this-panel'></button>
     <% end %>
 
-
-    <% if viewer.show_download? %>
+    <% if viewer.show_download_all? && viewer.purl_object.size > 1073741824 %>
```

Comment:
Can you put this in `app/views/footer/_file.html.erb` and then we can remove the `show_download_all?` method?    This will also require changing this line: https://github.com/sul-dlss/sul-embed/blob/305b3c12924a3de19040e8c96f3335e9e26ce013/app/views/embed/template/_file.html.erb#L6

Reply:
I thought about doing this too, will it be ok to duplicate the rest of `footer/_generic.html.erb` in`footer/_file.html.erb`?

Reply:
Yes, that's fine. If you wanted to make it really dry, you could extract the common parts out to a template included by both.

Reply:
I went with splitting off the template for now, without the extra template complexity.